### ISSUE TITLE
Title:Towards Non-saturating Recurrent Units for Modelling Long-term Dependencies

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#10994]
Title:Towards Non-saturating Recurrent Units for Modelling Long-term Dependencies<br>
Authors:Sarath Chandar, Chinnadhurai Sankar, Eugene Vorontsov, Samira Ebrahimi Kahou, Yoshua Bengio Abstract: Modelling long-term dependencies is a challenge for recurrent neural networks. This is primarily due to the fact that gradients vanish during training, as the sequence length increases.<br>
http://bit.ly/2EixgxW
